### PR TITLE
65700

### PR DESCRIPTION
**Pull request by himadree**

### CODE REVIEW

File: src/css/view/quick-view.scss [hunk 1]
```diff
@@ -577,6 +577,29 @@
     }
 }
 
+//Theme Supports
```
Comment (robinwpdeveloper) [inline src/css/view/quick-view.scss:580]: Change to `Theme Compatibility`

File: src/css/view/quick-view.scss [hunk 2]
```diff
@@ -577,6 +577,29 @@
...
+
+//Storefront Theme
+.theme-storefront {
+    .eael-product-popup {
+        form.cart.variations_form .variations td select {
+            padding: 6px;
+        }
```
Comment (robinwpdeveloper) [inline src/css/view/quick-view.scss:587]: Add 1 line spacing after each css block

File: src/css/view/quick-view.scss [hunk 3]
```diff
@@ -577,6 +577,29 @@
...
+        .woocommerce-product-gallery {
+            position: relative;
+            .woocommerce-product-gallery__trigger {
+                position: absolute;
+                top: 5px;
+                left: 5px;
+                z-index: 1;
+            }
+        }
```
Comment (robinwpdeveloper) [inline src/css/view/quick-view.scss:596]: 1 line spacing needed

File: src/css/view/woo-product-list.scss [hunk 4]
```diff
@@ -767,4 +767,15 @@ $spacer-lg: $spacing5;
...
     }
+}
+
+.theme-blocksy {
+    .eael-product-list-add-to-cart-button {
+        .button:before {
+            width: fit-content;
+            filter: none !important;
+            opacity: 1;
+            z-index: 0;
```
Comment (robinwpdeveloper) [inline src/css/view/woo-product-list.scss:778]: Can we use `z-index: 1`?

**Review by robinwpdeveloper — CHANGES_REQUESTED** [review]
Minor feedback added

File: src/css/view/woo-product-list.scss (hunk 4 above)
Comment (robinwpdeveloper) [reply src/css/view/woo-product-list.scss:778]: > Can we use `z-index: 1`?

@himadree bhai, how about this one? Should we update the z-index?

Comment (himadree) [reply src/css/view/woo-product-list.scss:778]: Bhai z-index should "0", to work properly.

**Comment by himadree** [comment]
All changes are done.

**Review by robinwpdeveloper — APPROVED** [review]

**Pull request merged**